mlrun/artifacts/manager.py

```diff
@@ -191,6 +191,8 @@ def log_artifact(
         if db_key is None:
             # set the default artifact db key
             if producer.kind == "run":
+                # In case of a run kind producer, we enable a key that is distinct from the db_key,
```

Comment:
do you mean, an artifact produced by a run?
also "enable" as "allow"?

Reply:
I meant if the producer kind is "run", and maybe it's better to say that we "create" a db_key differ from the key in order to allow.. ?